src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -132,9 +132,16 @@ public bool TryFetch(OptionKey optionKey, out object value)
                 }
                 else
                 {
-                    value = optionKey.Option.DefaultValue;
+                    value = null;
+                    return false;
```

Comment:
Nit: once you start adding early `returns`, I tend to prefer not having the trailing default case to ensure you think about each branch.